resin/settings.py

```diff
@@ -43,7 +43,9 @@ class Settings(object):
         # cache time : 1 week in milliseconds
         'image_cache_time': str(1 * 1000 * 60 * 60 * 24 * 7),
         # token refresh interval: 1 hour in milliseconds
-        'token_refresh_interval': str(1 * 1000 * 60 * 60)
+        'token_refresh_interval': str(1 * 1000 * 60 * 60),
+        # requests timeout: 30 seconds
+        'timeout': '30',
```

Comment:
nit: Is it worth being concerned about the other times in the settings being in milliseconds, but this time being in seconds?

Reply:
The requests library takes the timeout parameter as seconds.
I don't mind switching to milliseconds, it's up to you @nghiant2710 .

Reply:
Makes sense. Please switch to milliseconds @zvin